backend/app.py

```
@@ -0,0 +1,71 @@
+        if result.inserted_id:
+            return jsonify({
+                'success': True,
+                'message': 'To-Do item added successfully',
+                'item_id': str(result.inserted_id)
+            })
+        else:
+            return jsonify({
+                'success': False,
+                'message': 'Failed to add to-do item'
+            }), 500
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)